presidio-image-redactor/presidio_image_redactor/dicom_image_redactor_engine.py

```diff
@@ -567,27 +567,28 @@ def _make_phi_list(
         return phi_str_list
 
     @staticmethod
-    def _get_bboxes_from_analyzer_results(analyzer_results: list) -> dict:
+    def _get_bboxes_from_analyzer_results(analyzer_results: list) -> List[dict]:
         """Organize bounding box info from analyzer results.
 
         :param analyzer_results: Results from using ImageAnalyzerEngine.
 
         :return: Bounding box info organized.
         """
-        bboxes_dict = {}
-        for i in range(0, len(analyzer_results)):
+        bboxes = []
+        for i in range(len(analyzer_results)):
             result = analyzer_results[i].to_dict()
 
-            bboxes_dict[str(i)] = {
+            bbox_item = {
```

Comment:
isn't `result` already a dict? why do we need to copy it to a new one?

Reply:
I create a new one such that we only keep what is necessary for the downstream bounding box operations, and so that we don't alter the existing results in case the user wants to explore `analyzer_results` on their own separately.